step-api/step-api-keyword/src/main/java/step/handlers/javahandler/Input.java

```diff
@@ -0,0 +1,12 @@
+package step.handlers.javahandler;
+
+import java.lang.annotation.*;
+
+@Target({ElementType.PARAMETER, ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+@Inherited
+public @interface Input {
+	String name() default "";
+	String defaultValue() default "";
```

Comment:
Because java annotations only support the following return types, or a compile-time error occurs (https://docs.oracle.com/javase/specs/jls/se8/html/jls-9.html#jls-9.6.1):
* A primitive type
* String
* Class or an invocation of Class

* An enum type
* An annotation type

* An array type whose component type is one of the preceding types